db2i_charsetSupport.cc

```diff
@@ -80,7 +80,8 @@ static EncodingMapEntry encoding_map[] = {
   {"cp932", "IBM-943", "UTF-16", 1200}, // (SJIS for Windows Japanese)
   {"macce", "IBM-1282", "IBM-1153", 1153}, // (Mac Central European)
 
-  // There are no iconv converter tables for the following charsets:
+  // There are no iconv converter tables for the following charsets if specified
+  // DB2I_ERR_UNSUPP_CHARSET error is returned
```

Comment:
The comment above the table needs to be expanded. It looks like it's just the notes you jotted down when we were talking about it:

```
// single byte is ebcdic single byte
// multibyte is UTF-16
// Here is the list of supported mariadb charsets
// https://mariadb.com/kb/en/supported-character-sets-and-collations/
```

something like


```
// This table contains the mappings used to convert between MySQL/MariaDB
// charsets and Db2 CCSIDs. It tries to replicate the prior code which
// used the following mapping rules:
// - map single byte charsets to the matching EBCDIC CCSID
// - map the ucs2 charset to 13488
// - map the utf8 charset to 1208
// - map any other charset to 1200
//
// Instead of going through a bunch of code to try to map charsets to
// iconv names and then iconv names to names that QlgCvtTextDescToDesc
// understands to get the CCSID that we can pass to QTQGESP to get the
// associated CCSID... we do that work upfront and store it in this table.
// 
// NOTE: Any charset which is not in this list will cause the
// DB2I_ERR_UNSUPP_CHARSET error to be returned, however the Db2 CCSID does
// not have to match since existing tables may have been mapped differently.
```